examples/apps/tree-comparison/src/model/newTreeInventoryList.ts

```diff
@@ -37,6 +38,7 @@ export class InventorySchema extends builder.object("Contoso:Inventory-1.0.0", {
 	inventoryItemList: InventoryItemList,
 }) {}
 
+// eslint-disable-next-line import/no-deprecated
 export const treeConfiguration = new TreeConfiguration(
```

Comment:
it would be nice to update usages like these both to serve as examples of how to update usages, but also to test that we can update them and have it still work. Also avoiding having the tech debt of out of date API use and lint suppressions is nice.

Reply:
I already have this change partially staged in a different PR and would prefer to make it there. Updating the code is trivial (and does work), but this demo also has a lot of comments which I audited once for present-day applicability.

Reply:
Might be nice to link that (possible as draft) from the PR description as having it would help evaluate this change.